ivy/compiler/compiler.py

```diff
@@ -32,16 +32,16 @@ def compile(
     kwargs: Optional[dict] = None,
 ) -> Union[Graph, LazyGraph]:
     if python_version[1] == "8":
-        from ._compiler_38 import compile as _compile
+        from ._compiler_38 import trace as _trace
```

Comment:
Yep. We'll merge all the PRs later. So this will be merged after updating GC and generating the new  `.so` file. For testing, I'll use _compile for now 😄